PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalClient.kt

```diff
@@ -44,6 +42,11 @@ class PayPalClient internal constructor(
      */
     private var isVaultRequest = false
 
+    /**
+     * Used for sending Shopper Insights session ID provided by merchant to FPTI
+     */
+    private var shopperSessionId: String? = null
```

Comment:
Should we save this in a repository singleton outside of the Client class? Maybe a PayPalRepository that holds all PayPal specific values.

Reply:
I considered adding it to the `AnalyticsParamRepository` but since it is passed in by the merchant at each step there didn't seem much value there and it seemed a little confusing that we don't do that for any other general analytics event params. Open to a new param repository or adding it to the existing one if folks think that would be more helpful!

Reply:
Ah gotcha. That makes sense 👍